redbeams/src/main/java/com/sequenceiq/redbeams/repository/DatabaseServerConfigRepository.java

```diff
@@ -32,4 +32,6 @@
             @Param("names") Set<String> names,
             @Param("workspaceId") Long workspaceId,
             @Param("environmentId") String environmentId);
+
+    Optional<DatabaseServerConfig> findByClusterCrn(String stackCrn);
```

Comment:
Parameter should be clusterCrn, both here and in the service layer.

Reply:
Good catch, they are renamed